opentelemetry-api/src/opentelemetry/propagators/__init__.py

```diff
@@ -15,6 +15,18 @@
 """
 API for propagation of context.
 
+The propagators for the
```

Comment:
Nit: Can we have consistent line wrappings?

Reply:
Fixed